src/test/rustdoc-ui/issue-79495.stderr

```diff
@@ -0,0 +1,13 @@
+error: generic parameters may not be used in const operations
+  --> $DIR/issue-79495.rs:9:35
+   |
+LL |     Some(T) = std::mem::size_of::<T>(),
+   |                                   ^ cannot perform const operation using `T`
+   |
+   = note: type parameters may not be used in const expressions
+   = help: use `#![feature(generic_const_exprs)]` to allow generic const expressions
```

Comment:
I think a better test here would enable this feature -- that likely tests a larger section of code.

Reply:
Adding `#![feature(generic_const_exprs)]` still causes the same ICE, so I will close this pull request.